content/STRUCTURE.json

```diff
@@ -140,6 +140,12 @@
      "menu": ["LXD", "Getting started - OpenNebula"],
      "generator": "markdown",
      "meta": {"input": "lxd/getting-started-opennebula.md"}},
+     
+    {"path": "/lxd/advanced-guide/",
+     "title": "LXD - Advanced Guide",
+     "menu": ["LXD", "Advanced Guide"],
```

Comment:
I think this should be capitalized as "Advanced guide" for consistency

Reply:
Ok, I will change this.